front/src/Phaser/Login/CustomizeScene.ts

```diff
@@ -115,7 +116,7 @@ export class CustomizeScene extends AbstractCharacterScene {
             gameManager.setCharacterLayers(layers);
 
             this.scene.sleep(CustomizeSceneName);
-            gameManager.tryResumingGame(this, EnableCameraSceneName);
+            gameManager.tryResumingGame(this, localUserStore.wasCompanionSet() ? EnableCameraSceneName : SelectCompanionSceneName);
```

Comment:
I suggest a compromise: Show it only if the user selected "design your own [character]".

Reply:
I changed it, so that it is only accessible from the menu. Showing it only after the "design your own" scene also makes sense. But showing it to someone, who just want's to quickly jump into a meeting probably doesn't.